### DISCUSSION TITLE
Pane of Smart Connections is empty (only digits in top left) Worker sent an error (Console)

### DISCUSSION THREAD

**Discussion by Ramadas108** [brianpetro/obsidian-smart-connections#727] [General]
What console says. It happened after I changed local models for embedding. 

![Obsidian_2MnK7nDtrH](https://github.com/user-attachments/assets/96866e24-2729-4018-a843-b4a103a07ec5)

**Comment by Ramadas108** [comment]
I needed to remove Smart Connections, remove the hidden folder of smart connections and then it worked. 

Some bug happens when changing the model for embedding

**Comment by Cossackx** [comment]
Same thing happening to me now. Troubleshooting...
![image](https://github.com/user-attachments/assets/e2333ee7-81c3-434d-9a2f-abbf3eeb53bd)
